README.md

```diff
@@ -19,10 +19,12 @@ Next, open `Login Demo.xcworkspace` in XCode (not `Login Demo.xcodeproj`,
 since you need to include the Cocoapod dependencies).
 
 You'll then need to edit the file `Login Demo/ViewController.m` and specify the
-Firebase app you're using as well as your Twitter / Google OAuth API keys. To setup
+Firebase app you're using as well as your Twitter API key. To setup
 Facebook auth, you'll need to open `Supporting Files/Info.plist`.
 and set FacebookAppID and FacebookDisplayName as well as configure a
-URL Scheme to match your App ID.
+URL Scheme to match your App ID. To setup
+Google auth, you'll need to add a `GoogleService-Info.plist` as well as configure a
```

Comment:
Agreed, it's a confusing link too, so better that we link it that way rather than sticking it in the code.